docs/general/administration/hardware-acceleration/apple.md

```diff
@@ -0,0 +1,103 @@
+---
+uid: admin-hardware-acceleration-apple
+title: Apple Mac
+---
+
+# HWA Tutorial On Mac
+
+This tutorial guides you on setting up full video hardware acceleration on Apple Macs via VideoToolbox.
+
+## Acceleration Methods
+
+[VideoToolbox](https://developer.apple.com/documentation/videotoolbox) is the only available method on macOS.
+
+To achieve full acceleration, [Metal](https://developer.apple.com/metal/) is required.
+
+## Tone-mapping Methods
+
+Hardware accelerated HDR to SDR tone-mapping is supported on all Macs from 2017 and later, with the exception of the MacBook Air (13-inch, 2017).
+
+There are two different methods that can be used. Pros and cons are listed below:
+
+1. **Metal**
+
+   - Pros: Supports Dolby Vision P5, detailed fine-tuning options.
+
+   - Cons: Slower on low end GPUs, especially on Intel iGPUs.
+
+2. **VideoToolbox Native**
+
+   - Pros: Lower power consumption, less dependency on GPU performance, good visual quality without manual fine-tuning.
+
+   - Cons: Lack of tuning options, does not support Dolby Vision P5.
+
+When both methods are enabled, VideoToolbox Native will be used for most videos, and Metal will only be used as a fallback for Dolby Vision Profile 5 videos.
+
+## Select System Hardware
+
+Hardware accelerated transcoding is supported on all Macs that support [VideoToolbox](https://developer.apple.com/documentation/videotoolbox). This includes most 2011 and later Macs.<sup>*</sup>
+
+Full acceleration is available on most Macs from 2017 and later, with the exception of the MacBook Air (13-inch, 2017).
+
+To have native Apple Silicon support, Jellyfin server 10.9.0+ and `jellyfin-ffmpeg6` 6.0.1-5 or higher is required.
+
+While hardware acceleration via VideoToolbox might work on older series Macs, it is not officially supported.
+
+If your Mac does not have an internal display, you may need to connect it to a monitor or use an HDMI dummy plug to prevent the GPU from being throttled.
```

Comment:
```suggestion
If your Mac does not have an internal display, you may need to connect it to a monitor or use a dummy plug to prevent the GPU from being throttled.
```
There are dummy plugs for other display interfaces.

Reply:
Well I can accept this suggestions, but I have to say it is either HDMI or USB-C for desktop Macs after 2017 and the USB-C one cost more usually.